trikot-viewmodels-declarative-flow/compose-flow/src/main/kotlin/com/mirego/trikot/viewmodels/declarative/configuration/TrikotViewModelDeclarative.kt

```diff
@@ -12,7 +12,11 @@ object TrikotViewModelDeclarative {
             return internalImageProvider ?: throw uninitializedException
         }
 
-    fun initialize(imageProvider: VMDImageProvider) {
+    var textStyleProvider: VMDTextStyleProvider? = null
+        private set
+
+    fun initialize(imageProvider: VMDImageProvider, textStyleProvider: VMDTextStyleProvider? = null) {
```

Comment:
at this point, imageProvider could be optionnal too

Reply:
I like this suggestion. I can do a similar change to iOS in #152 if we decide to go ahead with this change.

Reply:
I am just not sure what advantage the dummy implementation would give. It would still not be able to resolve any resource. Is it to keep the force initialisation and throw exception if not initialize?

Reply:
🤔 Yeah, with default value it's harder to enforce initialization. But being able to initialize with (null, null) somehow feels wrong...

Reply:
okay, made the changes. Let me know if it looks good